corehq/apps/commtrack/util.py

```diff
@@ -122,20 +110,26 @@ def _needs_commtrack_config(domain):
                 caption='Stock-out',
             ),
         ],
-    )
-
-    if requisitions_enabled:
-        config.requisition_config = get_default_requisition_config()
+    ).save()
 
-    config.save()
 
-    program = get_or_create_default_program(domain.name)
+def _enable_commtrack_previews(domain):
+    for toggle_class in (
```

Comment:
Honestly it's a bit unclear.  I think we definitely don't want the reverse operation of this to be destructive, in case something happens like someone accidentally turning it off and turning it back on.  With this PR, all that'll happen is that `commtrack_enabled` and `toggles.COMMTRACK` will be flipped - you'll still have locations and the advanced vellum stuff.  I think that's appropriate, but could be talked out of it.